File: src/main/java/site/gachontable/gachontablebe/domain/pub/presentation/dto/request/PubRegisterRequest.java
package site.gachontable.gachontablebe.domain.pub.presentation.dto.request;

public record PubRegisterRequest(String pubName,
                                 String oneLiner,
                                 String instagramUrl,
                                 Integer minutes,
                                 String menuUrl,
                                 Boolean openStatus,
                                 Boolean waitingStatus) {
}

File: src/main/java/site/gachontable/presentation/pub/dto/request/PubRegisterRequest.kt
package site.gachontable.presentation.pub.dto.request;

public record PubRegisterRequest(String pubName,
                                 String oneLiner,
                                 String instagramUrl,
                                 Integer minutes,
                                 String menuUrl,
                                 Boolean openStatus,
                                 Boolean waitingStatus) {
}
